zerver/webhooks/intercom/fixtures/user_created.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Why did you delete `user_added` and created `user_created` instead?

Reply:
the even is triggered when user is create. More accurate wording.